src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/SynapseReactClientEntityQueryKey.java

```diff
@@ -8,19 +8,19 @@
 import jsinterop.annotations.JsType;
 
 @JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
-public class SynapseReactClientQueryKey {
+public class SynapseReactClientEntityQueryKey {
```

Comment:
Renamed because this query key structure currently only applies to the entity cache